src/scss/modules/table-list.scss

```diff
@@ -50,6 +50,10 @@
 
 .table-list__meta {
   margin-right: 8px;
+
+  &:empty {
+    margin-right: 0;
+  }
```

Comment:
There are times when the form button region (`[data-form-region]`) is empty but still has `margin-right: 8px` applied to it. That sometimes adds unnecessary extra margin when the attachments icon is being shown.

This stops that from happening.

Reply:
could also be `.table-list__meta:not(:empty)`

Reply:
Yup... definitely like that better. Updated in last push.